app/config.py

```diff
@@ -105,6 +118,13 @@ def _build_db_uri(self, ssl_mode, hide_password=False):
             db_uri += f"?sslmode={self._db_ssl_mode}&sslrootcert={self._db_ssl_cert}"
         return db_uri
 
+    def _from_dict(self, dict, name, default):
+        value = dict.get(os.environ.get(name, default))
+
+        if value is None:
+            raise ValueError(f"{os.environ.get(name)} is not a valid value for {name}")
+        return value
```

Comment:
```suggestion
        raw = os.environ.get(name, default)
        try:
            return dict[name]
        except KeyError as error:
            raise ValueError(f"{raw} is not a valid value for {name}. Valid values are {dict.keys()}.") from error
```

It’s easier to forgive than to allow. This is more pythonic.